content/minutes/cab/2022-04-20-minutes.md

```diff
@@ -0,0 +1,32 @@
+---
+date: "2022-04-20T00:00:00+00:00"
+title: "Apr 20, 2022 Advisory Board Meeting"
+tags: ["Open Force Field Consortium", "Advisory Board", "minutes"]
+categories: ["Advisory Board Meeting Minutes"]
+draft: false
+description: "Meeting minutes"
+weight: 10
+author: "Diego Nolasco"
+---
+
+The [Open Force Field Consortium Advisory Board](https://openforcefield.org/about/organization/#open-force-field-consortium) met on April 20, 2022.
+The minutes are summarized as follows:
+
+#### OpenFF Toolkit releases 0.10.4 and 0.10.5
+
+* J. Wagner announced that release 0.10.4 ensures that AM1BCC ELF10 is used for partial charge calculations if OpenEye is available.
+
+* J. Wagner annouced that release 0.10.5 contains a major refactor of virtual sites.
+
+#### Coordinating Toolkit Biopolymer Functionality with OpenFE
+
+* OpenFF and OpenFE are building a synergistic relationship with respect to Biopolymer Functionality. OpenFE developers are acting as users+developers making targeted improvements to our code to meet your needs.
+
+#### Virtual Sites into Rosemary
+
+* D. Nolasco announced that after a good number of technical analyses, the lead team states that it is not possible to link the Virtual Sites implementation to the Rosemary release. Anyway, even though this is not a higher priority item, the OpenFF team will continue to pursue its implementation, as well as all the necessary tests for its insertion as a Rosemary feature.
```

Comment:
```suggestion
* D. Nolasco announced that after a good number of technical analyses, the lead team states that it is not possible to guarantee that Virtual Sites will be in the Rosemary release. Research into protein parameters and virtual sites will proceed in parallel. If the timing works out then virtual sites will be added to the Rosemary release, but if adding virtual sites would lead to substantial delay then the Rosemary release will proceed without them.
```